internal/template/template.go

```diff
@@ -36,8 +36,8 @@ var mergoOpts = []func(*mergo.Config){
 // FlattenInfo describes keys to flatten. If index is not a nil pointer then it is assumed the
 // value is a list that must be flattened at the specific index.
 type FlattenInfo struct {
-	Key   string `json:"key"`
-	Index *int   `json:"index"`
+	Key   string `hcl:"key,attr" json:"key"`
```

Comment:
Is the hcl tag syntax documented anywhere?

Reply:
https://godoc.org/github.com/hashicorp/hcl2/gohcl

Reply:
added a comment